library/storm

```diff
@@ -1,5 +1,6 @@
 Maintainers: Elisey Zanko <elisey.zanko@gmail.com> (@31z4)
 GitRepo: https://github.com/31z4/storm-docker.git
+Architectures: amd64, arm32v5, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
```

Comment:
So, this won't actually work correctly -- arches like `arm32v6` aren't supported on Debian-based tags, so they'll always fail and arches like `arm32v7` aren't supported on Alpine-based tags, so they'll also always fail.  You can, however, have this global list for the most common list, and then only override per entry where you need a list that doesn't match the global (such as for Alpine-based tags).

... and now that I've actually taken a look, all your tags are based on the same `openjdk:8-jre-alpine` image, so you need this list to simply match the list of that tag:

```console
$ bashbrew cat openjdk:8-jre-alpine | grep Architectures
Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
```

(from https://github.com/docker-library/official-images/blob/ef965f2b8e738edcf14d75042c67f308ad8b7a47/library/openjdk#L114-L117)

Reply:
Thanks so much @tianon 🙏 Fixed.